.circleci/config.yml

```
@@ -1961,6 +1961,10 @@ commands:
               fi
             fi
 
+            if [[ "${CI_JOBNAME}" == "race-condition-tests" ]]; then
+              MAIN_IMAGE_TAG="$(make --quiet tag)-rcd"
```

I think you'll need to permafix the tag for other dependencies on `make tag` but if setting MAIN_IMAGE_TAG is enough then maybe not :)
```suggestion
              TAG="$(make --quiet tag)"
              echo "${TAG}-rcd" > CI_TAG
              make --quiet tag
```

---

Let's see cause nothing is "made" in this step and we are just using the premade things to deploy